kubernetes/PostTarget.cmake

```diff
@@ -0,0 +1,9 @@
+target_link_libraries(${pkgName} yaml websockets)
+set_target_properties(${pkgName} PROPERTIES LINKER_LANGUAGE C)
+
+option(BUILD_EXAMPLES "Build examples" OFF)
```

Comment:
I prefer that the `examples` folder is a separate CMake project. This fits the current folder structure better.

Could you remove this part and create an independent `examples/CMakeLists.txt` that will link `libkubernetes.{a|so}` ?

Reply:
yea sure I'll do that